neo4j/internal/errorutil/errors_test.go

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [https://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package errorutil_test
+
+import (
+	"fmt"
+	"github.com/neo4j/neo4j-go-driver/v5/neo4j/internal/errorutil"
+	"reflect"
+	"testing"
+)
+
+func TestCombineAllErrors(outer *testing.T) {
```

Comment:
nit: since you use `errorutil.CombineErrors` for asserting the correctness of `errorutil.CombineAllErrors` (which is perfectly valid), it'd be good to have tests for the former function as well to make sure the over all output is meaningful.

Reply:
yep, I took a shortcut yesterday since it was mostly a distraction to what I wanted to do, will take some more time today to add tests

Reply:
fixed in 3b403181f9a66276b68f7bba7858fea085229e2d